script/tasks/diamondSyncDEXs.sh

```diff
@@ -120,10 +120,10 @@ function diamondSyncDEXs {
         # call diamond
         if [[ "$DEBUG" == *"true"* ]]; then
           # print output to console
-          cast send "$DIAMOND_ADDRESS" "batchAddDex(address[])" "${PARAMS[@]}" --rpc-url "$RPC_URL" --private-key $(getPrivateKey "$NETWORK" "$ENVIRONMENT") --legacy
+          cast send "$DIAMOND_ADDRESS" "batchAddDex(address[])" "${PARAMS[@]}" --rpc-url "$RPC_URL" --account $(getAccount "$NETWORK" "$ENVIRONMENT") --legacy
```

Comment:
no --password flag needed?

same applies for all other cast send calls in this file